pages/linux/spi.md

```diff
@@ -0,0 +1,27 @@
+# spi
+
+> A meta package manager that handles both packages and slackbuilds.
```

Comment:
```suggestion
> A meta package manager that handles both packages and slackbuilds.
> More information: <https://linux.die.net/man/5/spi>.
```

Reply:
I actually added it to the pull request with the upstream url: https://github.com/gapan/spi

Reply:
Oh perfect! I couldn't find it.